installing_on_kubernetes/_topics/pods_ssl.md

```diff
@@ -0,0 +1,74 @@
+## Using TLS to encrypt connections between pods inside the cluster:
+
+### Create a secret containing all of the certificates
+The certificates should all be signed by a CA and that CA certificate should be in uploaded as `root_crt` so that it can be used to verify connection validity.  If the secret is named `internal-certificates-secret`, no changes are needed in the CR, if you choose a different name, that should be set in the `internalCertificatesSecret` field of the manageiq CR.
+
+        oc create secret generic internal-certificates-secret \
```

Comment:
Similar comment here but `triple-backtick sh`

Reply:
Also, only 4 space indent for code-blocks (othewise you end up with indenting inside the code as well, which makes copy-paste harder)...

Compare

    oc create secret generic internal-certificates-secret \

to

        oc create secret generic internal-certificates-secret \

Reply:
Done